test/test_autograd.py

```diff
@@ -8087,6 +8087,8 @@ def test_rnn_backward_to_input_but_not_parameters(self, device):
         out.sum().backward()
         self.assertFalse(s.grad is None or s.grad.abs().sum().item() == 0)
 
+    @unittest.skipIf(IS_ARM64, "ITT is not available on ARM")
```

Comment:
If we have `is_available()`, isnt `IS_ARM64` a redundant check?

Reply:
Yeah, I think so. Let's use `is_available()` then. I'll remove the `IS_ARM64` check.